packages/odyssey-react/src/components/Table/TableContainer.tsx

```diff
@@ -26,11 +35,11 @@ export type Props = {
    * The visible heading for the table
    */
   title: ReactNode;
-} & ComponentProps<"figure">;
+}
 
 type Ref = HTMLElement;
 
-const TableContainer = forwardRef<Ref, Props>((props, ref) => {
+let TableContainer = forwardRef<Ref, Props>((props, ref) => {
   const { children, title, ...rest } = props;
 
   const omitProps = useOmit(rest);
```

Comment:
lets move the spread expression for `omitProps` below on line 48 to be the first prop assigned to `figure` so we don't open up the possibility of extending props we want to control for in the future